QA: fix sticky position of Gallery block

nahbee10 opened the pull request:
### Description

<!--- Summarize the changes that can be found in this PR and how your reviewers should test these changes. -->
- remove `overflow-hidden` from Gallery container to fix not working sticky position setup for Gallery text 

### Type(s) of changes

<!--- Put an `x` in all the boxes that apply. -->

- [x] Bug fix
- [ ] New feature
- [ ] Update to an existing feature

### Motivation for PR

<!--- If it addresses an open issue, please link to the issue here, otherwise, briefly describe the issue. -->
QA item from Notion
- [the sticky effect from the live website has been broken](https://www.notion.so/garden3d/It-appears-the-sticky-effect-from-the-live-website-has-been-broken-in-the-preview-link-a1b7d642bfc4402188006e550509bc9a?pvs=4)

### How Has This Been Tested?

<!--- Please note how you have tested your changes. Browsers, accessibility, devices, unit tests, etc. -->
- Tested with [preview link](https://sanctu-dot-md7x363fb-sanctucompu.vercel.app/) on desktop browser

### Applicable screenshots

<!--- When appropriate, upload screenshots. -->
https://www.loom.com/share/5c5a0952ead14564b1840db82f1e3968

### Follow-up PR

<!--- When appropriate, please note what your reviewers can expect in a follow up PR. -->

mokaymokay reviewed (commented):
I think we added this to fix the issue of links overflowing horizontally, can we add `overflow-x-hidden` (maybe to Markdown?) to make sure that issue won't reoccur?

nahbee10 commented:
> I think we added this to fix the issue of links overflowing horizontally, can we add `overflow-x-hidden` (maybe to Markdown?) to make sure that issue won't reoccur?

good point! I implemented this by adding `overflow-x : hidden` to separate CSS file since any Basement overflow classes(hidden, scroll, auto) will disable sticky effect. Here's the [PREVIEW](https://sanctu-dot-hv6ga8wv4-sanctucompu.vercel.app/) for fixed version!

mokaymokay commented:
Alternatively, we can add it to Markdown as I've suggested:

<img width="1756" alt="Screen Shot 2023-02-10 at 5 30 24 PM" src="https://user-images.githubusercontent.com/30246503/218144626-bfb85bdd-e6d3-49ff-a90b-f01b58d107d3.png">

nahbee10 commented:
> Alternatively, we can add it to Markdown as I've suggested:
> 
@mokaymokay This doesn't solve the issue! I think the overflow should be prevented from more parent container.
<img width="470" alt="Screen Shot 2023-02-10 at 12 39 20 PM" src="https://user-images.githubusercontent.com/12539032/218161591-b4135b48-51da-47bd-9aa8-acf317b5f223.png">

nahbee10 commented on styles/config.scss:
```diff
@@ -19,3 +19,11 @@ $spacing-units: 0rem, 0.25rem, 0.325rem, 0.5rem, 0.625rem, 0.75rem, 0.9rem, 1rem
 
 $grid-columns: 8;
+
+$additional-class-definitions: (
+  'overflow-x-visible': (
+    overflow-x: visible,
+  ),
+);
+
+$additional-responsive-classes: (overflow-x-visible);
```
@mokaymokay I ended up with using custom basement definitions!

mokaymokay replied:
nice
